sdk/communication/azure-communication-identity/src/samples/java/com/azure/communication/identity/CreateAndIssueToken.java

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.communication.identity;
+
+import com.azure.communication.common.CommunicationUserIdentifier;
+import com.azure.communication.identity.models.CommunicationTokenScope;
+import com.azure.core.credential.AccessToken;
+import com.azure.core.credential.AzureKeyCredential;
+import java.util.Arrays;
+import java.util.List;
+
+public class CreateAndIssueToken {
+    public static void main(String[] args) {
+        String endpoint = System.getenv("AZURE_COMMUNICATION_ENDPOINT");
```

Comment:
Wouldn't it be better to use the connection string instead of the endpoint and access key as separate values? There should a parse_connection_string function somewhere that might help you extract both values. I'm saying this because we should ideally use the least amount of environment variables to make the process a little less confusing

Reply:
Also, now that we´re here... Have you thought about using the same name as the one we've been using for our connection string environment variable? The process of running the samples would be easier for us as we should already have that variable set up :)